targets/TARGET_Ambiq_Micro/TARGET_Apollo3/TOOLCHAIN_ARM_STD/AMA3B1KK.sct

```diff
@@ -40,7 +40,18 @@
 #define MBED_RAM0_SIZE  0x100
 #define MBED_RAM1_START (MBED_RAM0_START + MBED_RAM0_SIZE)
 #define MBED_RAM1_SIZE  (MBED_RAM_SIZE - (MBED_RAM0_SIZE))
+; This section does not contain any symbols. It is only used for the linker
+; to calculate the size of the stack sections and assign values to stack
+; symbols later
+#define STACK_DUMMY_START (MBED_RAM1_START + MBED_RAM1_SIZE)
+#define STACK_DUMMY_SIZE 0x8
```

Comment:
We got the info from the GCC linker script - the ranges of the stack and the heap are both shifted down by 8 bytes: https://github.com/ARMmbed/mbed-os/blob/3174a4c6a55c769f51a8fc6185a6fb4fd181d557/targets/TARGET_Ambiq_Micro/TARGET_Apollo3/TOOLCHAIN_GCC_ARM/AMA3B1KK.ld#L187

Dummy sections exist in many GCC linker scripts but most them do not shift any bytes. I wonder if the 8 bytes are used for something in particular on this target? Thanks in advance. @ARMmbed/team-sparkfun